### PR TITLE
Fixed v-model and added 'inputEmited' feature

### PR DESCRIPTION
**Pull request by VoIdJMp**

### CODE REVIEW

**Comment by fayzzzm** [comment]
@VoIdJMp names of commits and PR should be capitalized and should be at least meaningful.

File: .codesandbox/ci.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "sandboxes": ["new", "vanilla"]
```
Comment (fayzzzm) [inline .codesandbox/ci.json:2]: This is fixed, first, you should merge the first PR and pull it there, and should be OK.

File: src/components/VueMoney.js [hunk 2]
```diff
@@ -3,7 +3,13 @@ import { checkToOnlyNumber, moneyFormat } from "@utils/moneyUtils";
...
     name: "VueMoney",
     props: {
-        minMax: {
+        value: {
+            required: true,
+            type: [Number, String],
+            default: 0
+        },
+
+        min_max: {
```
Comment (fayzzzm) [inline src/components/VueMoney.js:12]: `min_max, oldValue` are two separate methodologies of naming. Either do `minMax` or `old-value`

File: src/components/VueMoney.js [hunk 3]
```diff
@@ -13,25 +19,36 @@ export default {
...
     },
+    watch: {
+        value: {
+            immediate: true,
+            handler (newValue) {
+                this.oldValue =  moneyFormat(newValue)
+                this.value = moneyFormat(newValue)
+            }
+        }
+    },
```
Comment (fayzzzm) [inline src/components/VueMoney.js:30]: nice

File: src/components/VueMoney.js [hunk 4]
```diff
@@ -13,25 +19,36 @@ export default {
...
     methods: {
-        onInput(event) {
-            const value = event.target.value.replaceAll(",", "");
+        onInput(e) {
```
Comment (fayzzzm) [inline src/components/VueMoney.js:32]: use event better

File: src/components/VueMoney.js [hunk 5]
```diff
@@ -13,25 +19,36 @@ export default {
...
                 this.oldValue = formatMoney;
             }
+            this.$emit('input', e.target.value)
         },
     },
-    render(createElement) {
-        return createElement("input", {
-            attrs: { type: "text", name: "money-input", class: "vf-money" },
-            on: { input: this.onInput },
+    render(h) {
```
Comment (fayzzzm) [inline src/components/VueMoney.js:48]: same for here, dont' use `h` use something else

**Review by fayzzzm — CHANGES_REQUESTED** [review]

File: src/components/VueMoney.js (hunk 4 above)
Comment (VoIdJMp) [reply src/components/VueMoney.js:32]: Ok

File: src/app.js [hunk 6]
```diff
@@ -15,18 +20,21 @@ const createTemplateByType = (type, createElement) => {
...
-        VueMoney: () => import("./components/VueMoney"),
+        VueMoney: () => import("./components/VueMoney.js"),
     },
     props: {
-        type: {
-            type: String,
-            default: TemplateType.MONEY,
-        },
+        type: { type: String, default: TemplateType.MONEY },
+        value: { required: true, type: [Number, String], default: 0 },
```
Comment (VoIdJMp) [inline src/app.js:27]: I suggest using this style of writing props (inline). 
I think it will be convenient.

**Pull request merged**